go/service/user.go

```diff
@@ -753,6 +753,31 @@ func (h *UserHandler) GetUserBlocks(ctx context.Context, arg keybase1.GetUserBlo
 	return res, nil
 }
 
+func (h *UserHandler) GetTeamBlocks(ctx context.Context, _ int) (res []keybase1.TeamBlock, err error) {
+	mctx := libkb.NewMetaContext(ctx, h.G())
+
+	defer mctx.TraceTimed(fmt.Sprintf("UserHandler#GetTeamBlocks()"), func() error { return err })()
```

Comment:
no need for `Sprintf` it there is no formatting
```suggestion
	defer mctx.TraceTimed("UserHandler#GetTeamBlocks()", func() error { return err })()
```